### ISSUE TITLE
.fireAncestors calls method on own component (and not on the parent)

### ISSUE THREAD

**Issue by michielvandergeest** [WebPlatformForEmbedded/Lightning#50]
Per documentation, `.fireAncestors` is a way to send a signal to a **parent** component. 

However, if a method with the same name exists on the component that is emitting the fireAncestors, that one is called instead (and the method on the parent is not called).

Example to reproduce:

```
export default class Foo extends lng.Component {

    static _template() {
        return {
            w: 1920,
            h: 1080,
            rect: true,
        }
    }

    _init() {
        this.fireAncestors('$bar')
    }

    $bar() {
        console.log('I get called, but I would expect $bar on my (distant) **parent** component to be called')
    }

}
```

**Comment by erikhaandrikman** [comment]
thanks for the report, we have a branch for the fix and are now testing it out, will notify you when it's in master.

https://github.com/WebPlatformForEmbedded/Lightning/tree/fix/fireAncestors

**Comment by michielvandergeest** [comment]
Confirmed that this fixes the issue 👍

**Comment by g-zachar** [comment]
Resolved by #51
